lib/ansible/modules/monitoring/zabbix/zabbix_host_info.py

```diff
@@ -177,6 +181,9 @@ def main():
             remove_duplicate=dict(type='bool', required=False, default=True),
             host_inventory=dict(type='list', default=[], required=False)
         ),
+        required_one_of=[
```

Comment:
This is a change of behavior. Right now, the module also works when none of these are specified, and returns `ok=False` and `hosts=[]` in that case. Not that this makes any sense, but it *could* be that a role or playbook relies on this behavior, so this can't be backported and requires a changelog fragment and potentially even a porting guide entry.

Reply:
I don't know what I was thinking, but it seems that I've completely forgot about `state: absent`. Thanks for noticing~